chromium_src/chrome/browser/icon_loader_win.cc

```diff
@@ -4,8 +4,8 @@
 
 #include "chrome/browser/icon_loader.h"
 
-#include <windows.h>
 #include <shellapi.h>
+#include <windows.h>
```

Comment:
This PR is failing `appveyor: win-x64-testing` here.

[Example](https://windows-ci.electronjs.org/project/AppVeyor/electron-i8wjp/build/1.0.989):

> In file included from ../../electron/chromium_src/chrome/browser/icon_loader_win.cc:7:
C:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\um\shellapi.h(70,1):  error: unknown type name 'DECLARE_HANDLE'
DECLARE_HANDLE(HDROP);
^
C:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\um\shellapi.h(73,1):  error: unknown type name 'EXTERN_C'
SHSTDAPI_(UINT) DragQueryFileA(_In_ HDROP hDrop, _In_ UINT iFile, _Out_writes_opt_(cch) LPSTR lpszFile, _In_ UINT cch);

Maybe this include order needs to be left as-is?

Reply:
I wanted to give it a shot.
It not only should be left as is, but also clang should "be aware" that they shouldn't be touched.